app/controllers/admin/data_sources_controller.rb

```diff
@@ -7,12 +7,18 @@ def show
       respond_to do |format|
         format.html
         format.csv do
-          send_data @data_source.meters.to_csv,
+          send_data @data_source.to_csv,
```

Comment:
If we no longer have a CSV download, then we don't need the controller method to respond to CSV.